app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
> When you say conversion step, you're talking about .toISOString()? It seems like SHARE results don't always have dates there.

This is defined in the `afterModel` hook, where `preprint` is the result of an API call- it doesn't seem like this specific usage of field data will ever be a SHARE object...? (see line 31) I might be missing something, so wanted to ask.

By conversion step, I mean the act of converting this field value into a date. By default, it seems that `DS.DateTransform` already converts things into a date for us.
`new Date(preprint.get('dateModified'))`